src/scripts/main.js

```diff
@@ -1,3 +1,53 @@
 'use strict';
 
-// write code here
+const field = document.querySelector('.field tbody');
+const appR = document.querySelector('.append-row');
```

Comment:
```suggestion
const appendRowButton = document.querySelector('.append-row');
```

Don't shorten names. Make them easy to understand.